src/components/ActionTooltip.vue

```diff
@@ -164,8 +170,11 @@ export default {
     name () { return this[`Name_${this.lang}`] },
     icon () { return `https://xivapi.com${this.Icon}` },
     actionCategory () { return this.ActionCategory ? this.ActionCategory[`Name_${this.lang}`] : '' },
+    actionNonCombatant () { return [6, 7].includes(this.ActionCategory.ID) },
```

Comment:
I hate magic numbers but ehhhhh

Reply:
Also this is inherently unsafe for lack of checking the ID exists but yolo

Reply:
put another `const` beneath `resourceMapping` for this. This sort of magic in the middle of a comprehension is a no-go IMO